src/gui/FileBrowser.cpp

```diff
@@ -338,6 +337,9 @@ FileBrowserTreeWidget::FileBrowserTreeWidget(QWidget * parent ) :
 
 }
 
+
+
+
```

Comment:
I recall somewhere about having 4 newlines between functions, but I don't know if we care to keep doing that? I say because this hunk is unrelated to PR, and more a formatting update.

Reply:
I wanted to keep FileBrowserTreeWidget's spacing a bit more consistent since it varied before, but I can undo this if you'd like.

Reply:
FWIW, I don't like the four newlines. (I'm generally in favour of anything that uses less vertical space.)

Reply:
I would also prefer a single newline between functions of the same class, but I've generalmy stuck with four in order to have some semblance of consistency.

Reply:
Yeah - I don't think it should be changed here, but it might be a candidate for the reformatting if others agree.

Reply:
I think it would be a thing for the clang-tidy/format PR.